docs/testlayer.rst

```diff
@@ -76,24 +76,29 @@ ZCMLLayer
 
 We now want a layer that loads up some ZCML from a file. The default
 is ``ftesting.zcml``, but here we'll load a test ``testlayer.zcml``.
+We can also choose to provide extra ZCML features (here, "devmode").
```

Comment:
That's what I wound up linking to:

> We can also choose to provide extra ZCML features that are used `to
 conditionally control processing of certain directives
 <http://zopeconfiguration.readthedocs.io/en/latest/narr.html#making-specific-directives-conditional>`_
 (here we use "devmode", a common condition for controlling development
 options like debugging output).